build.assets/makefiles/shrink-rootfs.mk

```diff
@@ -11,11 +11,16 @@
 # Disable lvm2 metadata daemon/socket and monitor
 # Disable block availability service (blk-availability.service) to avoid deactivation
 # of block devices on container stop
+#
+# TODO(r0mant): Disable *iscsi* services cause they may be running on host
+#               In the future we will need to enable them conditionally to
+#               be able to support OpenEBS cStor engine out of the box
 units := \
 	cgproxy.service cgmanager.service \
 	apt-daily.service apt-daily-upgrade.service \
 	lvm2-monitor.service lvm2-lvmetad.service lvm2-lvmetad.socket \
-	blk-availability.service
+	blk-availability.service \
```

Comment:
Systemd units are best disabled when the container is being built. You could use `systemctl mask <unit>` to do that.